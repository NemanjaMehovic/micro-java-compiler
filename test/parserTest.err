ERROR 15:22:24,900 - 
Sintaksna greska na liniji 5 simbol kod
ERROR 15:22:24,902 - Sintaksna greska prilikom definisanju promenljive na liniji 5
ERROR 15:22:24,905 - 
Sintaksna greska na liniji 8 simbol 1
ERROR 15:22:24,905 - Sintaksna greska prilikom nasledjivanja na liniji 8
ERROR 15:22:24,907 - 
Sintaksna greska na liniji 10 simbol y
ERROR 15:22:24,908 - Sintaksna greska prilikom definisanju promenljive na liniji 10
ERROR 15:22:24,914 - 
Sintaksna greska na liniji 23 simbol parametri
ERROR 15:22:24,915 - Sintaksna greska prilikom definisanja formalnih parametra funkcije na liniji 23
ERROR 15:22:24,919 - 
Sintaksna greska na liniji 30 simbol ;
ERROR 15:22:24,919 - Sintaksna greska kod if uslova na liniji 30
ERROR 15:22:24,920 - 
Sintaksna greska na liniji 32 simbol if
ERROR 15:22:24,921 - Sintaksna greska prilikom dodele vrednosti na liniji 4
ERROR 15:22:24,922 - Doslo je do greske prilikom parsiranja.
