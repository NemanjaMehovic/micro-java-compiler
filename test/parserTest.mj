

program parserTest

int greska kod globalne;
int x,y,z;

class a extends 1
{
    int x y,z;
    {
        a()
        {
            x = 1;
            y = 2;
            z = 3;
        }
    }
}

{

    int metoda(int losi parametri)
    {
        return 0;
    }

    void main()
    {
        if(;)
            y = 2;
        x = if;
    }
}