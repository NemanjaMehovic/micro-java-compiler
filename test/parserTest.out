INFO  15:22:24,829 - Compiling source file: C:\Users\neman\Desktop\PP_projekat\MJCompiler\test\test.mj
INFO  15:22:24,898 - #2 program
INFO  15:22:24,898 - #49 parserTest
INFO  15:22:24,898 - #49 int
INFO  15:22:24,899 - #49 greska
INFO  15:22:24,900 - #49 kod
INFO  15:22:24,900 - #49 globalne
INFO  15:22:24,900 - #38 ;
INFO  15:22:24,900 - #49 int
INFO  15:22:24,901 - #49 x
INFO  15:22:24,901 - #40 ,
INFO  15:22:24,904 - #49 y
INFO  15:22:24,904 - #40 ,
INFO  15:22:24,904 - #49 z
INFO  15:22:24,904 - #38 ;
INFO  15:22:24,904 - #4 class
INFO  15:22:24,904 - #49 a
INFO  15:22:24,904 - #11 extends
INFO  15:22:24,905 - #48 1
INFO  15:22:24,905 - #46 {
INFO  15:22:24,905 - #49 int
INFO  15:22:24,905 - #49 x
INFO  15:22:24,905 - #49 y
INFO  15:22:24,907 - #40 ,
INFO  15:22:24,907 - #49 z
INFO  15:22:24,907 - #38 ;
INFO  15:22:24,907 - #46 {
INFO  15:22:24,908 - #49 a
INFO  15:22:24,908 - #42 (
INFO  15:22:24,908 - #43 )
INFO  15:22:24,908 - #46 {
INFO  15:22:24,909 - #49 x
INFO  15:22:24,910 - #35 =
INFO  15:22:24,910 - #48 1
INFO  15:22:24,911 - #38 ;
INFO  15:22:24,911 - #49 y
INFO  15:22:24,912 - #35 =
INFO  15:22:24,912 - #48 2
INFO  15:22:24,912 - #38 ;
INFO  15:22:24,912 - #49 z
INFO  15:22:24,912 - #35 =
INFO  15:22:24,912 - #48 3
INFO  15:22:24,912 - #38 ;
INFO  15:22:24,912 - #47 }
INFO  15:22:24,912 - #47 }
INFO  15:22:24,913 - #47 }
INFO  15:22:24,913 - #46 {
INFO  15:22:24,914 - #49 int
INFO  15:22:24,914 - #49 metoda
INFO  15:22:24,914 - #42 (
INFO  15:22:24,914 - #49 int
INFO  15:22:24,914 - #49 losi
INFO  15:22:24,914 - #49 parametri
INFO  15:22:24,914 - #43 )
INFO  15:22:24,914 - #46 {
INFO  15:22:24,914 - #20 return
INFO  15:22:24,914 - #48 0
INFO  15:22:24,916 - #38 ;
INFO  15:22:24,916 - #47 }
INFO  15:22:24,916 - #10 void
INFO  15:22:24,917 - #49 main
INFO  15:22:24,917 - #42 (
INFO  15:22:24,917 - #43 )
INFO  15:22:24,918 - #46 {
INFO  15:22:24,918 - #16 if
INFO  15:22:24,918 - #42 (
INFO  15:22:24,919 - #38 ;
INFO  15:22:24,919 - #43 )
INFO  15:22:24,919 - #49 y
INFO  15:22:24,919 - #35 =
INFO  15:22:24,919 - #48 2
INFO  15:22:24,920 - #38 ;
INFO  15:22:24,920 - #49 x
INFO  15:22:24,920 - #35 =
INFO  15:22:24,920 - #16 if
INFO  15:22:24,921 - #38 ;
INFO  15:22:24,921 - #47 }
INFO  15:22:24,921 - #47 }
